### DISCUSSION TITLE
Consider adding back the "where is the public directory" table

### DISCUSSION THREAD

**Discussion by mkarajohn** [mswjs/mswjs.io#265] [Ideas]
When I first used the lib as a new user I thought [that section](https://v1.mswjs.io/docs/cli/init#where-is-my-public-directory) was very user friendly and thoughtful. I did not expect it to be gone in v2 docs, to be honest. 

@kettanaito consider adding it back maybe?

**Comment by TinsFox** [comment]
I think so too. We should add this part back.
